<documentation title="Trait name">
    <standard>
    <![CDATA[
    Traits MUST be suffixed by Trait: e.g. BarTrait.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: ">
        <![CDATA[
trait <em>BarTrait</em>
{
}
        ]]>
        </code>
        <code title="Invalid: ">
        <![CDATA[
trait <em>Bar</em>
{
}
        ]]>
        </code>
    </code_comparison>
</documentation>
